requirements/base.txt

```diff
@@ -5,80 +5,81 @@
 jdcal==1.4.1              # via openpyxl
 jinja2==2.11.2            # via coreschema
-l18n==2018.5              # via wagtail
+l18n==2020.6.1            # via wagtail
 markupsafe==1.1.1         # via jinja2
-mock==4.0.2               # via -r requirements/base.in
-mysqlclient==2.0.1        # via -r requirements/base.in
-newrelic==5.20.1.150      # via edx-django-utils
+mock==4.0.3               # via -r requirements/base.in
+mysqlclient==2.0.3        # via -r requirements/base.in
+newrelic==5.24.0.153      # via edx-django-utils
 oauthlib==3.1.0           # via requests-oauthlib, social-auth-core
 openapi-codec==1.3.2      # via django-rest-swagger
 openpyxl==3.0.5           # via tablib
-pbr==5.5.0                # via stevedore
+pbr==5.5.1                # via stevedore
 pillow==7.2.0             # via wagtail
-psutil==5.7.2             # via edx-django-utils
+psutil==5.8.0             # via edx-django-utils
 pycparser==2.20           # via cffi
-pycryptodomex==3.9.8      # via pyjwkest
+pycryptodomex==3.9.9      # via pyjwkest
 pyjwkest==1.4.2           # via edx-drf-extensions
 pyjwt[crypto]==1.7.1      # via drf-jwt, edx-auth-backends, edx-rest-api-client, social-auth-core
-pymongo==3.11.0           # via edx-opaque-keys
+pymongo==3.11.2           # via edx-opaque-keys
 python-dateutil==2.8.1    # via -r requirements/base.in, edx-drf-extensions
 python3-openid==3.2.0     # via social-auth-core
-pytz==2020.1              # via -r requirements/base.in, django, django-modelcluster, l18n
+pytz==2020.5              # via -r requirements/base.in, django, django-modelcluster, l18n
 pyyaml==5.3.1             # via edx-django-release-util
 requests-oauthlib==1.3.0  # via social-auth-core
-requests==2.24.0          # via coreapi, edx-drf-extensions, edx-rest-api-client, pyjwkest, requests-oauthlib, slumber, social-auth-core, wagtail
+requests==2.25.1          # via coreapi, edx-drf-extensions, edx-rest-api-client, pyjwkest, requests-oauthlib, slumber, social-auth-core, wagtail
 rest-condition==1.0.3     # via edx-drf-extensions
 semantic-version==2.8.5   # via edx-drf-extensions
 simplejson==3.17.2        # via django-rest-swagger
 six==1.15.0               # via cryptography, edx-auth-backends, edx-django-release-util, edx-drf-extensions, edx-opaque-keys, html5lib, l18n, pyjwkest, python-dateutil, social-auth-app-django, social-auth-core
 slumber==0.7.1            # via edx-rest-api-client
-git+https://github.com/python-social-auth/social-app-django.git@c00d23c2b45c3317bd35b15ad1b959338689cef8#egg=social-auth-app-django  # via -r requirements/github.in, edx-auth-backends
-social-auth-core==3.2.0   # via -c requirements/constraints.txt, -r requirements/base.in, edx-auth-backends, social-auth-app-django
-soupsieve==2.0.1          # via beautifulsoup4
+social-auth-app-django==4.0.0  # via -r requirements/base.in, edx-auth-backends
+social-auth-core==3.3.3   # via -r requirements/base.in, edx-auth-backends, social-auth-app-django
```

Comment:
This version isn't high enough to avoid the bug; it looks like the newer releases on PyPI were updated this weekend, maybe they had been temporarily removed when you ran this?

Reply:
Although this was the exact version on which the bug was fixed, I've upgraded to the latest version anyway.